content/blog/productivity-porn.md

```diff
@@ -0,0 +1,36 @@
+
+Each time I'd hear their message I would be hit with an electric shock when I realized that they were right. I would immediately shut off my phone. For about a week after I would steer clear of any and all productivity porn, but inevitably I would find it weaseling its way back into my life.
+
+I find productivity porn to be a uniquely terrifying negative effect of social media for four reasons. While these reasons are also relevant to the other negative impacts of social media, I think they are particularly fitting for productivity porn.
+
+- **It saps my intrinsic motivation.** When I find myself wasting excessive amounts of time on social media my intrinsic motivation normally acts as a back stop. Eventually I will become tired of scrolling on social media and will go do something productive. However, productivity porn short circuits this by making me think I was being productive the entire time.
+- **It is very deceptive.** As far as I can tell I've been negatively impacted by this since at least highschool and I'm only realizing it right now. Looking back on it, how much time have I actually spent doing things versus just thinking about doing them.
+- **It is necessary in moderation.** Unlike other parts of social media, you actually need some this in moderation. It is equally as bad to only ever do things and to never spend time learning about how to do them better. This makes avoiding it difficult because you can't just cut it out cold turkey.
+- **It is found in all forms of media.** This is the scariest reason for me. Typically when I feel like social media is too negative for me I feel safe running away to slower, less connected, and more deliberate forms of media --- books, podcasts, blog posts etc. These lack the virality of _social_ media platforms and mitigate the issues that arise. However, I find productivity porn is just as prevalent in traditional media as it is in social media. While perhaps it is more toned down when found in traditional media, it means that there is no safe haven from productivity porn.
+
+I've spent a lot of time thinking about how to conclude this post and I don't think I've come up with anything satisfying. The only thing that is clear to me is that it is abundantly important that I figure out how to stymie the negative effect that productivity porn is having in my life. When I look for solutions to this problem I only come up with half-baked ideas and more questions. Is this something you've encountered in your life? What have you done about it?

```

Comment:
```suggestion
I've spent a lot of time thinking about how to conclude this post and I don't think I've come up with anything satisfying. The only thing that is clear to me is that it is abundantly important that I figure out how to stymie the negative effect that productivity porn is having on my life. When I look for solutions to this problem I only come up with half-baked ideas and more questions. Is this something you've encountered in your life? What have you done about it?

```
effect _on_ your life, rather than effect _in_ your life, right? kinda on the fence about this one but I think I'm right?

Final note: I like your end call-to-action by asking that question, but there's no direct, frictionless way for the reader to reach out. Maybe add a hyperlink saying, "I'd love to hear." or "Reach out here." or something like that. Just a nit suggestion, not necessary.